corehq/apps/domain/tests/test_delete_domain.py

```diff
@@ -879,6 +887,29 @@ def test_users_delete(self):
         self._assert_users_counts(self.domain.name, 0)
         self._assert_users_counts(self.domain2.name, 1)
 
+    def test_users_domain_membership(self):
+        web_user = WebUser.create(self.domain.name, f'webuser@{self.domain.name}.{HQ_ACCOUNT_ROOT}', '******',
+                                  created_by=None, created_via=None)
+
+        another_domain = Domain(name="another-test", is_active=True)
+        another_domain.save()
+        self.addCleanup(another_domain.delete)
+
+        # add more than 1 domain membership to trigger _log_web_user_membership_removed
```

Comment:
```suggestion
        # add more than 1 domain membership to trigger _log_web_user_membership_removed in tests
```

Reply:
Thanks for the suggested change @snopoke .

since this is a test file, it should already be clear that it is being done for the test?

Reply:
I don't think that makes it any clearer why you need more than 1 membership. If I had just read this in the code I would have wondered why you need more than 1 to trigger logging